ETC.tsv

```diff
@@ -7938,8 +7938,8 @@ ETC_20150317_007938	Quest Buff
 ETC_20150317_007939	Move to Fallen Worry Forest through Gate Route and Spine Heart Dale
 ETC_20150317_007940	Move to Leaf Veil Plateau
 ETC_20150317_007941	I will move to Fedimian
-ETC_20150317_007942	Get leather from Jukopus
-ETC_20150317_007943	Get leather from Vubbe Thief
+ETC_20150317_007942	Get leather from Jukopus'
+ETC_20150317_007943	Get leather from Vubbe Thiefs
```

Comment:
Thiefs is grammatically incorrect. Either change it back to Vubbe Thief or Vubbe Thieves.